### ISSUE TITLE
Modelling Time: cyclic time, time ordinal eras

### ISSUE THREAD

**Issue by smrgeoinfo** [protegeproject/swrlapi#60]
The discussion on [Modelling time](https://github.com/protegeproject/swrlapi/wiki/ModellingTime) is very good, thanks!  I just wanted to point out that there are other views of time that are important in some scientific pursuits:
Cyclic time-- e.g. 'samples were collected annually on the equinox',  'gravity observations were made during the full moon',  'fireflies were collected at sunset', 'gauge measurements were made daily at high-tide'

Time ordinal eras-- the actual coordinates on a timeline are unknown, uncertain, or subject to debate, but the sequence of events is known.  e.g. the bison was killed by late Paleolithic hunters, Early Medieval walls were constructed on top of the Roman Era fortification, the sediments were deposited during the Inanigutta (Upper Cambrian) radiolarian stage.

**Comment by martinjoconnor** [comment]
Yup - the article does really seem to imply that there is only one view. I will update. 

Do you have any links to descriptions of the additional views?

**Comment by smrgeoinfo** [comment]
https://www.w3.org/TR/owl-time/ is a good place to have a look.  The informative references are a good place to get into the background literature.
